osquery/tables/system/posix/sudoers.cpp

```diff
@@ -61,6 +62,20 @@ void genSudoersFile(const std::string& filename,
       continue;
     }
 
+    // if last line contains a backslash as the last character,
+    // treat current line as part of previous line and
+    // append it to appropriate column.
+    if (isLongLine) {
+      isLongLine = (line.at(line.size() - 1) == '\\');
+      if (results.back()["rule_details"].empty()) {
+        results.back()["header"].pop_back();
+      } else {
+        results.back()["rule_details"].pop_back();
+      }
+      results.back()["rule_details"].append(line);
```

Comment:
nit: Prefer storing the result of `results.back()` in a variable first, for instance `const auto& last_line = results.back()` instead of repeating `results.back()`.